From 4c0e7347109523ab64a29697dbec1ebf9b4c58c3 Mon Sep 17 00:00:00 2001
From: Konrad Leszczynski <konrad.leszczynski@intel.com>
Date: Thu, 17 Mar 2016 10:09:05 +0000
Subject: [PATCH 6/7] usb: hub: rename u1_is_enabled and u2_is_enabled
 variables

Variables will be renamed to u1_allowed and u2_allowed
respectively. This way, the code will be more readable
and the values will reflect their functionality better.

Change-Id: Ic1c3726a91561d964a4213c9c8f16451d0011874
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-18376
Signed-off-by: Konrad Leszczynski <konrad.leszczynski@intel.com>
Reviewed-on: https://android.intel.com:443/482867
---
 drivers/usb/core/hub.c  |  4 ++--
 drivers/usb/core/hub.h  |  8 ++++----
 drivers/usb/core/port.c | 26 +++++++++++++-------------
 3 files changed, 19 insertions(+), 19 deletions(-)

diff --git a/drivers/usb/core/hub.c b/drivers/usb/core/hub.c
index 629ea8a..8b8f64d 100644
--- a/drivers/usb/core/hub.c
+++ b/drivers/usb/core/hub.c
@@ -4027,10 +4027,10 @@ void usb_enable_lpm(struct usb_device *udev)
 	}
 	port_dev = hub->ports[udev->portnum - 1];
 
-	if (port_dev->u1_is_enabled)
+	if (port_dev->u1_allowed)
 		usb_enable_link_state(hcd, udev, USB3_LPM_U1);
 
-	if (port_dev->u2_is_enabled)
+	if (port_dev->u2_allowed)
 		usb_enable_link_state(hcd, udev, USB3_LPM_U2);
 }
 EXPORT_SYMBOL_GPL(usb_enable_lpm);
diff --git a/drivers/usb/core/hub.h b/drivers/usb/core/hub.h
index 7317287..de57967c 100644
--- a/drivers/usb/core/hub.h
+++ b/drivers/usb/core/hub.h
@@ -84,8 +84,8 @@ struct usb_hub {
  * @portnum: port index num based one
  * @power_is_on: port's power state
  * @did_runtime_put: port has done pm_runtime_put().
- * @u1_is_enabled: whether u1 should be enabled.
- * @u2_is_enabled: whether u2 should be enabled.
+ * @u1_allowed: whether u1 should be allowed.
+ * @u2_allowed: whether u2 should be allowed.
  */
 struct usb_port {
 	struct usb_device *child;
@@ -96,8 +96,8 @@ struct usb_port {
 	u8 portnum;
 	unsigned power_is_on:1;
 	unsigned did_runtime_put:1;
-	unsigned u1_is_enabled:1;
-	unsigned u2_is_enabled:1;
+	unsigned u1_allowed:1;
+	unsigned u2_allowed:1;
 };
 
 #define to_usb_port(_dev) \
diff --git a/drivers/usb/core/port.c b/drivers/usb/core/port.c
index 2b7fed8..57bb749 100644
--- a/drivers/usb/core/port.c
+++ b/drivers/usb/core/port.c
@@ -54,13 +54,13 @@ static ssize_t usb3_lpm_show(struct device *dev,
 	struct usb_port *port_dev = to_usb_port(dev);
 	const char *p;
 
-	if (port_dev->u1_is_enabled) {
-		if (port_dev->u2_is_enabled)
+	if (port_dev->u1_allowed) {
+		if (port_dev->u2_allowed)
 			p = "u1_u2";
 		else
 			p = "u1";
 	} else {
-		if (port_dev->u2_is_enabled)
+		if (port_dev->u2_allowed)
 			p = "u2";
 		else
 			p = "0";
@@ -78,20 +78,20 @@ static ssize_t usb3_lpm_store(struct device *dev,
 	struct usb_hcd *hcd;
 
 	if (!strncmp(buf, "u1_u2", 5)) {
-		port_dev->u1_is_enabled = true;
-		port_dev->u2_is_enabled = true;
+		port_dev->u1_allowed = true;
+		port_dev->u2_allowed = true;
 
 	} else if (!strncmp(buf, "u1", 2)) {
-		port_dev->u1_is_enabled = true;
-		port_dev->u2_is_enabled = false;
+		port_dev->u1_allowed = true;
+		port_dev->u2_allowed = false;
 
 	} else if (!strncmp(buf, "u2", 2)) {
-		port_dev->u1_is_enabled = false;
-		port_dev->u2_is_enabled = true;
+		port_dev->u1_allowed = false;
+		port_dev->u2_allowed = true;
 
 	} else if (!strncmp(buf, "0", 1)) {
-		port_dev->u1_is_enabled = false;
-		port_dev->u2_is_enabled = false;
+		port_dev->u1_allowed = false;
+		port_dev->u2_allowed = false;
 	} else
 		return -EINVAL;
 
@@ -239,8 +239,8 @@ int usb_hub_create_port_device(struct usb_hub *hub, int port1)
 	port_dev->power_is_on = true;
 	port_dev->dev.parent = hub->intfdev;
 	if (hub_is_superspeed(hdev)) {
-		port_dev->u1_is_enabled = true;
-		port_dev->u2_is_enabled = true;
+		port_dev->u1_allowed = true;
+		port_dev->u2_allowed = true;
 		port_dev->dev.groups = port_dev_usb3_group;
 	} else
 		port_dev->dev.groups = port_dev_group;
-- 
1.9.1

